co-log/src/Colog/Message.hs

```diff
@@ -491,6 +501,11 @@ builderDmyHMSz (C.Datetime date time) =
 -- Utility functions
 ----------------------------------------------------------------------------
 
+{- | Shows a thread id in the following format:
+
+>>> showThreadId <$> Control.Concurrent.myThreadId
+"[ThreadId 4898] "
```

Comment:
I'm afraid this test won't work because `ThreadId` is non-deterministic and could be different each time 😞 
But I wonder how tests are passing here on multiple CIs 🤔

Reply:
Seems `co-log` doesn't run `doctest` - I'll take a look what changes are required to add it.

Reply:
See https://github.com/co-log/co-log/pull/228